caffeine@patapon.info/extension.js

```diff
@@ -948,6 +979,13 @@ class Caffeine extends QuickSettings.SystemIndicator {
             this._manageScreenBlankState(true); // Allow blank screen
             this._manageNightLight(false, true);
             this.addInhibit(appId); // Inhibit app
+
+            // Uninhibit previous focused apps
+            this._appInhibitedData.forEach((data, id) => {
+                if (id !== appId) {
+                    this.removeInhibit(id);
+                }
```

Comment:
This is a bugfix right ? We were keeping inhibitors for apps that were previously in focus ?

Reply:
Yes, before we were keeping previously inhibitors for apps in focus mode, but as soon as user focus a none trigger app, then all of them were removed.
With the new subtitle label that show which app is enabling Caffeine, I wanted to update the app name when focusing two trigger app one by one. For that, I need to remove the previous inhibitor.

I just realize that the 'user' inhibitor is also removed and that's not good because when having an active timer and focusing an app, it disable caffeine...
I will write a separate commit to fix this situation.